client/lib/jetpack/wpcom-upsell-controller.tsx

```diff
@@ -23,27 +21,24 @@ export default function upsellSwitch(
 		const siteId = getSelectedSiteId( getState() );
 		const isJetpack = isJetpackSite( getState(), siteId );
 
+		// Note that isJetpack *includes* Atomic sites, so we don't need to explicitly check for that.
 		if ( isJetpack || isEnabled( 'jetpack-cloud' ) ) {
 			return next();
 		}
 
-		const currentPlan = getSitePlan( getState(), siteId );
-		const isBusiness = isBusinessPlan( currentPlan.product_slug );
-
 		// Show Business plan upsell if it isn't a Jetpack site and it doesn't have
-		// a Business plan
-		if ( ! isBusiness ) {
+		// a Business plan. Note this will include E-Comm.
```

Comment:
Aren't E-Comm plans by default Jetpack sites? My understanding is that installing WooCommerce triggers the Automated Transfer, so there shouldn't be cases of E-Comm plan sites getting down here.

Reply:
That makes sense, but also there may be edge cases we're not aware of. In the end, I decided it's better to stick with an existing selector that covers Biz and E-Comm plans. Also if we ever have an AT offering in addition to the Biz plan it's best to not hard-code things if we don't have to?

Reply:
I agree 100%. This is me trying to confirm with you the things I'm learning on each day :)